File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int el = INT_MIN;
        for(int i : nums){
            if(i == el) return true;
            el = i;
        }

        return false;
    }
};

File: Data Structures & Algorithms/remove-element/submission-0.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int wp = 0;
        for(int i = 0; i < nums.Length;i++){
            if(nums[i] != val){
                nums[wp] = nums[i];
                wp++;
            }
        }
        return wp;
    }
}